### DISCUSSION TITLE
v2023.06.11-release

### DISCUSSION THREAD

**Discussion by dougtinkham** [Theriak-Domino/theriak-domino#5] [Announcements]
2023.06.11 Release Notes

This release is a bugfix release that fixed 1 bug introduced late in the 
2023.06.06 release.

Binaries are provided below for macos Apple Silicon (TheriakDominoMACas.2023.06.11),
macos Intel (TheriakDominoMACi.2023.06.11), Windows (TheriakDominoWIN.2023.06.11), 
and Ubuntu (TheriakDominoLIN.2023.06.11).

A more complete list of changes are found in the CHANGELOG.md text file in 
the root directory, along with older release notes in Documentation/history. 

Instructions for installation of programs and program use are in the 
Documentation/readme file. 

Detailed explanations of the programs and their general use are in 
Documentation/TheriakDominoGuide.pdf. 

Basic instructions for building programs from source using updated makefiles 
for Mac, Linux and Windows can be found on the main page of the GitHub site:
https://github.com/Theriak-Domino/theriak-domino

Updated versions of databases and ax model translations are and will continue 
to be posted on the Databases GitHub site: https://github.com/Theriak-Domino/Databases

2023.06.11

<hr /><em>This discussion was created from the release <a href='https://github.com/Theriak-Domino/theriak-domino/releases/tag/v2023.06.11-release'>v2023.06.11-release</a>.</em>

**Comment by rhumhun** [comment]
Thanks and congratulations for the new release and for putting Theriak Domino on github.
I have a new MacSilicon and I installed gfortran with the a standalone installer (https://github.com/fxcoudert/gfortran-for-macOS/releases). In this case the path for the library libquadmath.a in located here : /usr/local/gfortran/lib/libquadmath.a 

I used the makefile to compile. It works but neither theriag, theriakd, theriaq, theriak3 nor therieq are compiled (object file, no executale). I changed the makefile by adding the missing programm in all, as following:
all : theriak domino guzzler explot therbin therter thermo thalia makemap plotxy jfiles jphases \
 cat2 therion pretherion theriakd theriak3 theriaq theriag

can you help me? Thanks a lot
Cheers
Romain

**Reply by dougtinkham** [reply]
Building in static mode on macos with the gfortran compiler requires specifying the path to libquadmath.a, as indicated in the makefile. The path is currently set for the location when it is installed using Homebrew on an Apple Silicon mac, and if you install to a different location it does need modified, as you did.

TheriaG and TheriakD are no longer distributed with Theriak Domino so they were removed from the all target. I will remove the individual program build targets in the next release. Theriaq is not completely tested and ready for general use so it is not included in the all target, but it can be built as it has a target defined for it. Theriak3 and therieq appear to have been removed as well (I believe they were for internal use and not planned for general distribution). The all target contained all programs that were planned for distribution at this stage.

Cheers
Doug
